automation/requirements.txt

```diff
@@ -1,11 +1,11 @@
 cryptography<3.4
-pulpcore==3.14.0
+pulpcore==3.14.1
 pulp-file==1.8.1
 pulp-container==2.7.0
 pulp-deb==2.13.0
-pulp-rpm==3.13.2
+pulp-rpm==3.13.3
```

Comment:
Looks like this reflects https://github.com/theforeman/pulpcore-packaging/commit/eb2a83e5734b05a240d700aedd11aabcd9d04964.

Reply:
Correct